BEIMA.Backend.Test/DeviceFunctions/AddDeviceTest.cs

```diff
@@ -24,13 +31,38 @@ public async Task NoDevice_AddDevice_ReturnsValidId()
                   .Verifiable();
             MongoDefinition.MongoInstance = mockDb.Object;
 
+            // Setup storage provider.
+            var storageProvider = StorageProviderExtensions.CreateAzureStorageProvider();
```

Comment:
Thinking about it, I actually think we should try and use mock storage providers for these unit tests. We should probably only have unit tests for the actual storage providers in the storage provider unit tests themselves.

Reply:
I agree, just like how the mongo connector is a mock object

Reply:
Sounds good, will get that switched to using the mocker.